tools/deploymentsSchemaTests/TestSchemaCache.cs

```diff
@@ -0,0 +1,166 @@
+﻿using Azure.Deployments.Core.Components;
+using Azure.Deployments.Core.Extensions;
+using Azure.Deployments.Core.Resources;
+using Azure.Deployments.Templates.Contracts;
+using Azure.Deployments.Templates.Export;
+using Azure.Deployments.Templates.Extensions;
+using Azure.Deployments.Templates.Helpers;
+using Azure.Deployments.TemplateSchemas;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace deploymentsSchemaTests
```

Comment:
I think we should stick to the .NET convention of pascal-case for namespace / assembly names. ("deploymentsSchemaTests" -> "DeploymentsSchemaTests")